src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -252,15 +255,16 @@ protected static LSP.MarkupContent CreateMarkupContent(LSP.MarkupKind kind, stri
 
             var item = new LSP.VSCompletionItem()
             {
-                FilterText = insertText,
+                TextEdit = textEdit,
                 InsertText = insertText,
-                Label = insertText,
-                SortText = sortText ?? insertText,
```

Comment:
Per the LSP spec, SortText and FilterText should default to label (instead of insertText). Just changing the test behavior to reflect this.